### ISSUE TITLE
Issue with Tolino E-Reader

### ISSUE THREAD

**Issue by valfost** [Born2Root/Fast-Font#21]
Hi, I don't know if anyone knows the e-reader brand Tolino but it's pretty big in Germany. 

All three fonts are displayed in the menu but every time I use them the first letters of a word are not written in bold. But their name is shown correctly. (in their intended format) Can anyone help me?

![bild](https://github.com/user-attachments/assets/8b188a89-bb84-440e-b74d-3dd7765dec95)

**Comment by Vlajd** [comment]
Experiencing the same issue on Tolino Shine, displays to some default serif font instead.

**Comment by delacor** [comment]
also got this problem.. :(

**Comment by delacor** [comment]
EDIT: It works, when you import a PDF to TXT. But its not fun to read it without any formatting

**Comment by Born2Root** [comment]
If it works with plain TXT documents, than the file format could be a reason.
PDFs have often embedded Fonts, so that they look the same on every device.

Have you already tried it with EPUB eBooks and is it the same behaviour like with PDFs?

**Comment by delacor** [comment]
Yes, I only got EPUBs on my Tolino. Apart from that one PDF I tested with

**Comment by Born2Root** [comment]
Ok. Thanks for the fast reply.
You could clarify the following to narrow down the problem:

Are the EPUBs DRM protected from the store or downloaded with Calibre?

On Kindle there was also an issue, where the fonts only worked with DRM books. 
In this case, those purchased directly from the Amazon store.
Local books pushed to the Kindle via Calibre doesn't allow the font change
Removing the DRM or converting them to AZW3/mobi unlocked the font change feature, maybe also a solution for Tolino.

There was also a user that wrote a little python script to apply the fast-reading feature directly in Calibre. Thats an alternative way, as for kindle that was a general solution. Maybe it also works for Tolino.
Here is the link to his script repo.
[https://github.com/giutala/calibre_bionic_converter](https://github.com/giutala/calibre_bionic_converter)

**Comment by orangebutblue** [comment]
I only have non-DRM epubs. It doesn't work with any of them.

**Comment by Born2Root** [comment]
As I mentioned, on Kindle it only worked with DRM books. Maybe thats the same for Tolino.

If you are using non DRM you can try the python Calibre script I linked in the previous comment.

**Comment by sixseben29-ux** [comment]
The script may work but I have no Idea how to run it if it is even possible since I don't own a computer and just my android phone
